src/components/PriceBar/TileColorBoard.tsx

```diff
@@ -70,27 +96,32 @@ const TileColorBoard: React.FC = () => {
     <>
       <Flex height="64px">
         <Center
-          width="100%"
+          width="55%"
```

Comment:
Any reason why change these flex styles? Seems not looking good at this point or its just me?
<img width="1839" alt="image" src="https://user-images.githubusercontent.com/12379895/190388325-289bd3c4-bc32-4686-a563-567bc8870b21.png">

Reply:
OK! I'll check it, thanks!